mobile-first.html

```diff
@@ -67,7 +67,7 @@ <h2 class="recent">My Recent Projects</h2>
   
             
           <div class="head-work">
-            <img src="images/Img Placeholder.png" alt="project-cover">
+            <img class="placeholder" src="images/Img Placeholder.png" alt="project-cover">
              <h2 class="story">Multi-Post Stories</h2>
              <p class="dailypost">A daily selection of privately personalized<br> reads; no accounts or sign-ups required.<br> has been the industry's standard dummy<br> text ever since the 1500s, when an<br> unknown printer took a standard dummy<br> text.</p>
             <ul class="techstack-container">
```

Comment:
- [ ] Great job so far :muscle: As the previous reviewer suggested, kindly ensure that the margin around your project card matches that of the Figma design. I noticed that there is still a bigger margin around it compared to the Figma design. Kindly refer to the design guidelines and fix this so your design looks more like the given design :100: 

 - [ ] Secondly, the previous reviewer also recommended that you make all the elements in that card ie. (image, title, text, button, list of technologies) have the same consistent horizontal margin on the left so they can be better aligned and look more professional :+1: 
 - From the image below you can see that the `<h2 class="story">Multi-Post Stories</h2>` is not properly aligned with the rest of the elements. Please fix this as well :wink:  

- Your Design
![image](https://user-images.githubusercontent.com/102699958/221247905-00197eed-6bcc-4267-b106-526f19746703.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/221247967-57663aa9-67a9-49d2-9b67-1209181d5bf0.png)